redis/tests/test_sentinel.rs

```diff
@@ -93,12 +98,7 @@ fn test_sentinel_read_from_multiple_replicas() {
             .get_connection()
             .unwrap();
 
-        assert_eq!(
-            redis::cmd("MGET")
-                .arg(&["{x}key1", "{x}key2"])
-                .query(&mut replica_con),
-            Ok(("foo".to_string(), b"bar".to_vec()))
-        );
+        check_replica_role_and_master(&mut replica_con, &master_client);
```

Comment:
you're checking that all of the connections are replicas of the correct master, but you're not testing the "multiple" part of the test name - you're not verifying that these are connections to separate replicas.

Reply:
Good point, so I just changed this to check that we actually connect to all replicas. I've also changed the number of replicas because there was only one. In the SentinelClient test I only get the connection (no client, so no address), so I couldn't do the same. But I'm thinking of replacing ConnectionInfo instead of Client as you suggested, this will probably allow me to improve the SentinelClient test too.